app/android/fastlane/Appfile

```diff
@@ -1,10 +1,3 @@
-json_key_file(ENV["JSON_KEY_FILE"]) # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
-package_name("au.edu.faims.fieldmark") # e.g. com.krausefx.app
+json_key_file(ENV["ANDROID_JSON_KEY_FILE"]) # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
+package_name(ENV["ANDROID_APP_ID"])
```

Comment:
In one place you provided a fall back default value - is that necessary as I note it was not provided here?

Reply:
Maybe we want it to fail if the app ID is not supplied?

Reply:
It does fail if package_name is not set with a suitable warning.

Reply:
On the default, I guess it doesn't hurt. The build would succeed if it is not set but the upload would fail.